cohttp-lwt-jsoo.opam

```
@@ -27,6 +27,7 @@ depends: [
   "cohttp" {= version}
   "cohttp-lwt" {= version}
   "lwt" {>= "3.0.0"}
+  "lwt_ppx" {with-test}
   "js_of_ocaml" {>= "3.3.0"}
```

```suggestion
  "conf-npm" {with-test}
  "js_of_ocaml" {>= "3.3.0"}
```
Libraries using `node` for tests generally use `conf-npm` to ensure node is installed

---

Examples are https://github.com/mnxn/jsonoo and https://github.com/mnxn/promise_jsoo

---

Well I learned something, thanks :)